cypress/integration/new_pipeline_page_test.js

```diff
@@ -47,7 +47,6 @@ describe('New Pipeline page tests', () => {
 
     it('correctly displays all blocks in its category in the visual editor', function () {
       cy.get('.main-content .flow-editor').within(() => {
-        cy.get('.block-item').should('have.length', this.blocks.data.length);
```

Comment:
instead of removing this entirely, we could add a constant to the length. We still want to make sure all received blocks are displayed together with the static ones

Reply:
You're right. I left the check with `this.block.data.length + 5`.